integration-test/extensionconfig/public-azurechina.json

```diff
@@ -0,0 +1,26 @@
+{
+	"docker" : {
+		"port" : "2376",
+		"options" : [
+			"--label",
+			"foo=bar"
+			]
+	},
+	"compose" : {
+		"envdump" : {
+			"image" : "mirror.azure.cn:5000/library/busybox",
```

Comment:
@lizzha shouldn't this be https? How come this non-TLS hostname is trusted in docker-engine even though it's not specified as `--insecure-registry` in `"options"`?